src/.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": "../.eslintrc.json",
+    "parserOptions": {
+        "tsconfigRootDir": "src",
+        "project": "./tsconfig-base.json"

```

Comment:
Are there any other rules that require the `project` option besides the ones we moved to `.eslintrc.ci.json`?

Reply:
@uniqueiniquity There were two custom rules, I refactored them and removed `project` option from default configs